checklists/checklistmap.php

```diff
@@ -131,8 +171,23 @@ function openIndPU(occId){
 			<?php echo (isset($LANG['MAYBE_RARE'])?$LANG['MAYBE_RARE']:'It may be that the vouchers have rare/threatened status that require the locality coordinates be hidden'); ?>.
 		</div>
 		<?php
-	}
+   } 
 	?>
 	<div id='map_canvas'></div>
+   <div class="service-container" 
+      data-voucher-coords="<?= htmlspecialchars(json_encode($coordArr, 4)) ?>"
+      data-checklist-meta="<?= htmlspecialchars(json_encode($clMeta, 2)) ?>"
```

Comment:
Indentation is just for readability. If you meant intention it is to separate the php and js as much as possible so they are less coupled. Like instead of pulling the php straight into js I encode the php variables as data attributes the html so they can be accessed via query or document selection.